Dockerfile

```diff
@@ -68,6 +68,8 @@ RUN pip install wheel \
   && mv .local/bin/bandit .local/bin/bandit2 \
   && pip3 install --user bandit==${BANDIT_VERSION}
 
+ENV SLITHER_VERSION 0.8.3
+RUN pip3 install --user --no-cache-dir slither-analyzer==${SLITHER_VERSION}
```

Comment:
Can you please expand on the use of `--user` and `--no-cache-dir` options in the `pip3 install` command? I am curious to understand what their purposes are. 🤔

Reply:
I copied this command from the spike.
The other `pip3` commands in Dockerfile also use `--user --no-cache-dir`.

`--no-cache-dir` seems to store some intermediate installation files.  According to this [stackoverflow post](https://stackoverflow.com/questions/45594707/what-is-pips-no-cache-dir-good-for), the cache is useless in a Docker image, and you can shrink the image size by with `--no-cache-dir`.